package.json

```diff
@@ -28,10 +28,11 @@
     "@react-navigation/native": "^6.1.1",
     "@react-navigation/native-stack": "^6.9.6",
     "@supabase/supabase-js": "^2.2.1",
-    "expo": "~47.0.8",
+    "expo": "^47.0.13",
     "expo-app-loading": "~2.1.1",
     "expo-font": "~11.0.1",
     "expo-image-picker": "~14.0.2",
+    "expo-linear-gradient": "^12.1.1",
```

Comment:
I'm getting a warning about this package not being compatible with the expo version, not super pertinent though